model/gof-metrics.wl

```diff
@@ -30,13 +30,68 @@ goodnessOfFitMetrics[residuals_,longData_]:=Module[{RmsRelativeError,MeanRelativ
 	deathsMask=(#[[1]]==1)&/@longData;
 	positivePcrMask=(#[[1]]==2)&/@longData;
 	<|
-		"chiSquared"->chiSquared[fit["FitResiduals"],longData],
-		"rmsRelativeError"->rmsRelativeError[fit["FitResiduals"],longData],
-		"rmsRelativeErrorDeaths"->rmsRelativeError[fit["FitResiduals"],longData,deathsMask],
-		"rmsRelativeErrorPcr"->rmsRelativeError[fit["FitResiduals"],longData,positivePcrMask],
-		"meanRelativeErrorDeaths"->meanRelativeError[fit["FitResiduals"],longData,deathsMask],
-		"meanRelativeErrorPCR"->meanRelativeError[fit["FitResiduals"],longData,positivePcrMask],
-		"rSquaredDeaths"->rSquared[fit["FitResiduals"],longData,deathsMask],
-		"rSquaredPCR"->rSquared[fit["FitResiduals"],longData,positivePcrMask]
-	|>
+		"chiSquared"->chiSquared[residuals,longData],
+		"rmsRelativeError"->rmsRelativeError[residuals,longData],
+		"rmsRelativeErrorDeaths"->rmsRelativeError[residuals,longData,deathsMask],
+		"rmsRelativeErrorPcr"->rmsRelativeError[residuals,longData,positivePcrMask],
+		"meanRelativeErrorDeaths"->meanRelativeError[residuals,longData,deathsMask],
+		"meanRelativeErrorPcr"->meanRelativeError[residuals,longData,positivePcrMask],
+		"rSquaredDeaths"->rSquared[residuals,longData,deathsMask],
+		"rSquaredPcr"->rSquared[residuals,longData,positivePcrMask]
+	|>/.InterpolatingFunction[___]->((0)&)
```

Comment:
we should add a todo to clean these up (on both datasets). We had thought that doing 3 day avg would be good -- but that might not be smooth enough to transition well to the constant values... I tried to use a spline quickly instead of an interpolation, but couldn't get the right incarnation of expanding the parametric variable over the time axis